stable/aws-for-fluent-bit/README.md

```diff
@@ -56,6 +56,7 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `cloudWatch.logFormat` | An optional parameter that can be used to tell CloudWatch the format of the data. A value of json/emf enables CloudWatch to extract custom metrics embedded in a JSON payload. See the Embedded Metric Format. |  |
 | `cloudWatch.roleArn` | ARN of an IAM role to assume (for cross account access). |  |
 | `cloudWatch.autoCreateGroup` | Automatically create the log group. Valid values are "true" or "false" (case insensitive). | true |
+| `cloudWatch.newLogGroupTags` | Tags to include with auto created log groups. Map of key/value pairs. | {} |
```

Comment:
Can you edit this slightly to make it even more clear- what you have right now is good, but let's say explicitly it only works if you set auto_create_group true and the log group did not already exist.

Reply:
Thanks - that's now done!